pkg/3scale/amp/operator/base_apimanager_logic_reconciler.go

```diff
@@ -82,3 +83,11 @@ func (r *BaseAPIManagerLogicReconciler) updateResource(obj common.KubernetesObje
 	r.Logger().Info(fmt.Sprintf("Updated object %s", ObjectInfo(obj)))
 	return r.Client().Update(context.TODO(), obj) // don't wrap error
 }
+
+func (r *BaseAPIManagerLogicReconciler) reconcilePodDisruptionBudget(desiredPDB *v1beta1.PodDisruptionBudget) error {
+	if r.apiManager.Spec.PodDisruptionBudget != nil && r.apiManager.Spec.PodDisruptionBudget.Enabled {
```

Comment:
It would be nice to have in `pkg/apis/apps/v1alpha1/apimanager_types.go`
```
func (apimanager *APIManager) IsPDBEnabled() bool {                              
  return apimanager.Spec.PodDisruptionBudget != nil && apimanager.Spec.PodDisruptionBudget.Enabled  
}                                                                                             
```

Then use it in any place:
```
if r.apiManager.IsPDBEnabled() {
}
```

Reply:
yeah, good idea. Will do that.

Reply:
Resolved